Spigot-Server-Patches/0438-Optimise-random-block-ticking.patch

```diff
@@ -527,7 +527,7 @@ index 0fed337130..c74b85917a 100644
 +                int randomZ = (location >>> 4) & 15;
  
 -                        Fluid fluid = iblockdata.getFluid();
-+                BlockPosition blockposition2 = blockposition.setValues(j + randomX, randomY, k + randomZ);
++                BlockPosition blockposition2 = blockposition.setValues(j + randomX, randomY, k + randomZ).immutableCopy();
```

Comment:
Missing required paper comments

```suggestion
+                BlockPosition blockposition2 = blockposition.setValues(j + randomX, randomY, k + randomZ).immutableCopy(); // Paper
```

Reply:
It's already inside paper start/end comments. See the bigger picture ;)

Reply:
Ah goddamnit, sorry :P